views/layouts/main.handlebars

```diff
@@ -15,8 +15,9 @@
     
     <!-- CSS Stylesheet -->
     <link rel="stylesheet" href="/assets/css/style.css" type="text/css" />
-
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
```

Comment:
Isn't there a jQuery reference a few lines above this?

Reply:
yeah got it i'll remove it